### PR TITLE
Update RELEASING.md update repo section

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: Andrew Leonard <anleonar@redhat.com>

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -242,22 +242,44 @@ At some point in a java version's lifecycle, the JDK version will be maintained
...
 
-1. Update the [configurations](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/jobs/configurations)
+1. ci-jenkins-pipelines: Update the [configurations](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/jobs/configurations)
 Rename the nightly build targets file (it will be named `jdkxx.groovy`, [example here](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk15u.groovy)) to be `jdkxxu.groovy`. Do the same for the pipeline config file (named `jdkxx_pipeline_config.groovy`, [example here](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk15u_pipeline_config.groovy)).
 
-2. Update the `javaToBuild` from `jdkxx` to `jdkxxu` inside the [pipeline job](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/build) that is being shifted to an update repository.
+2. ci-jenkins-pipelines: Update version from `jdkxx` to `jdkxxu` inside [docs/generateBuildMatrix.sh](Update https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/docs/generateBuildMatrix.sh)
```
Comment (M-Davies) [inline RELEASING.md:260]:
```suggestion
2. ci-jenkins-pipelines: Update version from `jdkxx` to `jdkxxu` inside [docs/generateBuildMatrix.sh](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/docs/generateBuildMatrix.sh)
```

**Review by M-Davies — CHANGES_REQUESTED** [review]
One small nitpick

**Review by M-Davies — APPROVED** [review]
Thanks for this! Looks good :)

File: RELEASING.md [hunk 2]
```diff
@@ -242,22 +242,44 @@ At some point in a java version's lifecycle, the JDK version will be maintained
...
 Rename the nightly build targets file (it will be named `jdkxx.groovy`, [example here](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk15u.groovy)) to be `jdkxxu.groovy`. Do the same for the pipeline config file (named `jdkxx_pipeline_config.groovy`, [example here](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk15u_pipeline_config.groovy)).
 
-2. Update the `javaToBuild` from `jdkxx` to `jdkxxu` inside the [pipeline job](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/build) that is being shifted to an update repository.
+2. ci-jenkins-pipelines: Update version from `jdkxx` to `jdkxxu` inside [docs/generateBuildMatrix.sh](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/docs/generateBuildMatrix.sh)
+
+3. ci-jenkins-pipelines: Run on "linux" platform your updated docs/generateBuildMatrix.sh script to generate the updated README.md table to be updated at the end of [README.md](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/README.md#build-status)
+
+4. ci-jenkins-pipelines: PR "run tests".
```
Comment (adamfarley) [inline RELEASING.md:264]: I know what this means, and you know what this means, but I think it should be clearer.

How about: "In the Pull Request, post a comment saying "run tests". This should execute tests a, b, and c against your changes. Monitor their progress via the links automatically added to the PR, and ensure they all pass.

File: RELEASING.md [hunk 3]
```diff
@@ -242,22 +242,44 @@ At some point in a java version's lifecycle, the JDK version will be maintained
...
+
+5. openjdk-build: Update the `JDKXX_VERSION` from `jdkxx` to `jdkxxu` inside the [build script constants](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/sbin/common/constants.sh) that is being shifted to an update repository.
+
+6. openjdk-build: Update the version from `jdkxx` to `jdkxxu` inside [.github/workflows/build.yml](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/.github/workflows/build.yml)
+
+7. openjdk-build: PR "run tests".
```
Comment (adamfarley) [inline RELEASING.md:270]: Ditto.

**Review by karianna — DISMISSED** [review]
+1 to Adam's comments

File: RELEASING.md (hunk 2 above)
Comment (andrew-m-leonard) [reply RELEASING.md]: I've removed this bullet as it's not needed, the changes should be dealt with like any other github change...

File: RELEASING.md (hunk 3 above)
Comment (andrew-m-leonard) [reply RELEASING.md]: ditto

**Review by adamfarley — APPROVED** [review]
LGTM

**Review by gdams — APPROVED** [review]

**Pull request merged**